src/main/resources/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ExtensibleChoiceParameterDefinition/config.jelly

```diff
@@ -31,34 +31,7 @@ THE SOFTWARE.
     <f:entry title="${%Description}" field="description" help="/help/parameter/description.html">
         <f:textarea />
     </f:entry>
-    <f:dropdownList title="${%Choice Provider}" name="choiceListProvider" help="${descriptor.getHelpFile('choiceListProvider')}">
-        <j:forEach
-                var="curDescriptor"
-                items="${descriptor.choiceListProviderList.iterator()}"
-                varStatus="loop">
-            <j:scope>
-            <j:set
-                var="choiceListProvider"
-                value="${curDescriptor==instance.choiceListProvider.descriptor?instance.choiceListProvider:null}"
-            />
-            <f:dropdownListBlock
-                title="${curDescriptor.displayName}"
-                value="${loop.index}"
-                selected="${choiceListProvider != null}"
-                staplerClass="${curDescriptor.clazz.name}"
-            ><j:scope>
-                <!-- Shown only when corresponding ChoiceProvider is selected. -->
-                <j:set var="descriptor" value="${curDescriptor}" />
-                <j:set var="instance" value="${choiceListProvider}" />
-                <!--
-                    Call config.jelly of the ChoiceProvider subclass.
-                    Do nothing if config.jelly is not provided.
-                -->
-                <st:include page="config.jelly" class="${curDescriptor.clazz}" optional="${true}" />
-            </j:scope></f:dropdownListBlock>
-            </j:scope>
-        </j:forEach>
-    </f:dropdownList>
+    <f:dropdownDescriptorSelector title="${%Choice Provider}" field="choiceListProvider" descriptors="${descriptor.enabledChoiceListProviderList}" />

```

Comment:
Gets to use dropdownDesciptorSelector to resolve [JENKINS-28046](https://issues.jenkins-ci.org/browse/JENKINS-28046).
And `descritor.enabledChoiceListProviderList` hides disabled providers.